### ISSUE TITLE
[helpdesk] Catalog artifacts and guidance documentation in preparation for Platform CMS Helpdesk

### ISSUE THREAD

**Issue by BerniXiongA6** [department-of-veterans-affairs/va.gov-cms#15580]
## Problem Statement
As someone who manages or assists with helpdesk user support, I need access to a catalog of clear and current processes and procedures in a centralized VA-approved tool or repository so that I can carry out Tier 1 helpdesk duties to CMS users. 

## What You Need to Know
- Some helpdesk artifacts are saved in a restricted, non-VA approved tool
- Our source of truth, GitHub, has outdated helpdesk artifacts
- There are many sources containing helpdesk artifacts that have competing and conflicting guidance (are outdated)

## Acceptance Criteria
- [ ] a worksheet or similar tool has been created for cataloguing the all the outdated and current helpdesk artifacts and guidance 
- [ ] Product and Delivery received access to the catalog of curated helpdesk artifacts and guidance 

## Tasks for Product
Berni and Marcia to write a ticket to include these other steps that need to happen before we move to Platform:
- a central location has been identified as the source of truth for housing helpdesk artifacts and guidance documentation (e.g. GitHub CMS repo)
- outdated processes and procedures have been updated within the last 30 days
- non-authorized VA tools have been deprecated or deactivated (e.g. Notion, Trello, etc.)

**Comment by BerniXiongA6** [comment]
@BerniXiongA6: Note to self, this ticket might be TOO big. Can pre-refine with @stefaniefgray and @TroyA6 in a near future sprint.

**Comment by stefaniefgray** [comment]
### Background info:
Notion replaced approximately 26 different Google Docs which were previously used for internal helpdesk documentation (no exaggeration, it was a lot to deal with!)

I created our Notion workbook around June/July 2022 to streamline our documentation before @TroyA6 joined our team so that he would not have to deal with keeping all of those tabs open at once and having to flip in between them all.

Happy to export elsewhere, but would like to explore VA-approved options and find the best fit. 📝 

One major benefit of Notion aside from the handy, user-friendly visual organization is that certain paragraphs of text or pre-scripted answers could be referenced across multiple pages -- and if changed in one location, it would automatically update across all locations. Super helpful feature!

Unsure if Confluence can do this? 🤔 Also, perhaps there are other VA-approved documentation tools out there that we might be unaware of, such as GitLab? 

Would love to see if we could obtain a full listing of approved documentation tools and go from there.

### Some export options:
- https://www.notion.so/help/category/import-export-and-integrate
- https://tiiny.host/blog/notion-export-how-to-export-your-data-from-notion/

### What to do with helpdesk artifacts:
- We'll need to catalog everything that's currently floating around Github, Confluence, etc. and determine what needs to be updated, replaced, or removed altogether -- probably good to put in a separate ticket.
- This might end up turning into an epic, but we've got this! 💪

**Comment by BerniXiongA6** [comment]
Thanks @stefaniefgray for the updates here. @maortiz-27-80 and I can work on creating and editing tickets for the epic. This should be a priority for helpdesk given that we want to make sure helpdesk guidance is updated for whomever will win the CMS recompete in Dec. 2023. cc: @TroyA6

**Comment by maortiz-27-80** [comment]
@TroyA6 - Please leave a comment on the progress you've made on this ticket.  I believe you mentioned you did some work on this sprint.  @stefaniefgray - We'll keep this ticket open and rollover into Sprint 99 with the unfinished work to be then completed. cc: @BerniXiongA6

**Comment by TroyA6** [comment]
I have started the cataloging process [here](https://docs.google.com/spreadsheets/d/1HrzBomlQWEA1yQ-Z_L8UIIdQKC0iZ0GzfBLq6AdM1MI/edit?usp=sharing).

Current status: I have documented the Document Name, Content Description, Document Format, and Current File Location for about 15 documents with more to come.

During this process, I have found that many documents contain links to other documents - I am trying to catalog without duplicating documents.

**Comment by stefaniefgray** [comment]
@TroyA6 Looks great so far! I'm editing and updating now -- adding dates of document creation and when each one was last updated, and also searching for other docs to throw in

**Comment by TroyA6** [comment]
@stefaniefgray - Perfect! Thank you!

**Comment by stefaniefgray** [comment]
@EWashb Confluence looks like a great place to move everything, and it's way better organized for the amount of written documentation that goes into our helpdesk work than Github. 👍 

@BerniXiongA6 @maortiz-27-80 @TroyA6 I've exported all of our Notion docs to Markdown files, but manually moving it to Confluence (if approved) will take some heavy focus time and will require a new ticket. Can discuss at sprint review today. Thanks!

**Comment by EWashb** [comment]
@stefaniefgray @BerniXiongA6 @maortiz-27-80 if you already have it in GitHub that's great. I'll defer to Berni and Marcia with how/what they would suggest for confluence.

**Comment by stefaniefgray** [comment]
@EWashb @BerniXiongA6 @maortiz-27-80 Oh it's not in Github yet! I should've clarified -- I meant that if we're transferring everything from Notion (exported as Markdown files) to a VA-approved platform, Confluence beats Github for the destination. :) Nothing is migrated yet since it will take manual work

**Comment by TroyA6** [comment]
@BerniXiongA6 @maortiz-27-80 - Regarding the "Product and Delivery received access to the catalog of curated helpdesk artifacts and guidance" - who is included in 'Product and Delivery'? and how is access provided? 

Regarding the preferred platform, will that be Confluence? If not, which platform is preferred?

**Comment by stefaniefgray** [comment]
Here's a link to where we've catalogued everything: https://docs.google.com/spreadsheets/d/1HrzBomlQWEA1yQ-Z_L8UIIdQKC0iZ0GzfBLq6AdM1MI/edit?usp=sharing (High-five @TroyA6 -- thank you so much for your teamwork and collaboration! You rock 🎉 )

I think we can count this ticket as "Done" for this sprint and jump on over into a new ticket to transfer all the Notion docs into Confluence (as well as the other tasks listed in the "Tasks for Product" section listed above).

Looking forward to seeing which spin-off tickets we'll need once Product and Delivery finalize next steps 📚 

cc: @BerniXiongA6 @EWashb @maortiz-27-80

**Comment by TroyA6** [comment]
@BerniXiongA6 @stefaniefgray @EWashb @maortiz-27-80  - I am closing this ticket per Stefanie's comment/confirmation above. Thank you all!
